### ISSUE TITLE
Packer build of Amazon Linux 2 fails

### ISSUE THREAD

**Issue by ThomasObenaus** [MatthiasScholz/cos#49]
Calling `packer build -var 'aws_region=us-east-1' -var 'ami_regions=us-east-1' nomad-consul-docker-ecr.json`

Fails with:

```
==> amazon-linux-ami2: Error modify AMI attributes: InvalidAMIAttributeItemValue: Invalid attribute item value " " for userId item type.
==> amazon-linux-ami2:  status code: 400, request id: 672be7fd-f89b-49ba-a76c-426341955d8d
```

**Comment by ThomasObenaus** [comment]
Packer version: 1.3.3

**Comment by ThomasObenaus** [comment]
# Root Cause
- `ami_users` can be filled with a list of strings
- strings like "xyz,abc" are automatically interpolated to a list of strings
- thus ["xyz,abc"] would create a nested list like [["xyz","abc"]]

**Comment by ThomasObenaus** [comment]
Solved by #50.
